### ISSUE TITLE
alt text is missing

### ISSUE THREAD

**Issue by Mu7ammadAbed** [GSG-G9/code-SOZAbit#28]
https://github.com/GSG-G9/code-SOZAbit/blob/83ab0b5ce6d5be8a1c29b6a4d32156db344560c0/index.html#L91
https://github.com/GSG-G9/code-SOZAbit/blob/83ab0b5ce6d5be8a1c29b6a4d32156db344560c0/index.html#L101
https://github.com/GSG-G9/code-SOZAbit/blob/83ab0b5ce6d5be8a1c29b6a4d32156db344560c0/index.html#L110

Each image should have an alt text. Not just for SEO purposes, but also because blind and visually impaired people won’t otherwise know what the image is about.
If you think the image doesn't have any purpose and doesn't need an alt then you can simply write it like that 
```html
<img src="images/edit.svg" alt="">
```
